sources/packages/backend/libs/integrations/src/institution-integration/ece-integration/ece-files/ece-file-header.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  DATE_FORMAT,
+  ECERequestFileLine,
+  ECE_SENT_TITLE,
+  RecordTypeCodes,
+  SPACE_FILLER,
+} from "../models/ece-integration.model";
+import { StringBuilder } from "@sims/utilities";
+
+const ORIGINATOR_CODE = "SSB";
+
+/**
+ * Header of a ECE request file.
+ * The documentation about it is available on the document
+ * 'SIMSSFAS - Institution File layouts In Analysis Folder'.
+ */
+export class ECEFileHeader implements ECERequestFileLine {
+  transactionCode: RecordTypeCodes;
```

Comment:
`transactionCode` was used for the CRA because it uses transaction code.
This one (and others from the past) can be named accordingly (e.g. `recordType`).

![image](https://user-images.githubusercontent.com/61259237/210871807-28bf8c1c-9e5d-4983-a239-34968ec12670.png)